src/callbacks_step/analysis_dg1d.jl

```diff
@@ -61,7 +61,10 @@ function calc_error_norms(func, u, t, analyzer,
                                        inv.(view(inverse_jacobian, :, element)))
 
         # Calculate errors at each analysis node
-        @. jacobian_local = abs(jacobian_local)
+        #@. jacobian_local = abs(jacobian_local)
+        for j in jacobian_local
+            j = abs(j)
+        end
```

Comment:
This is necessary to get LoopVectorization to work.

Reply:
I believe this will do nothing. What you need is something like this:
```suggestion
        #@. jacobian_local = abs(jacobian_local) # does not work with LoopVectorization
        for (index, value) in enumerate(jacobian_local)
            jacobian_local[index] = abs(value)
        end
```

Reply:
~~This is necessary to get LoopVectorization to work.~~

This is a solution that does not crash should be more accurate - you are probably right that loop vectorization does not become active with this fix